[2023-03-05T20:10:40.148+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: formula1_upload_to_s3.results_upload manual__2023-03-05T20:10:38.818733+00:00 [queued]>
[2023-03-05T20:10:40.158+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: formula1_upload_to_s3.results_upload manual__2023-03-05T20:10:38.818733+00:00 [queued]>
[2023-03-05T20:10:40.158+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-03-05T20:10:40.158+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 6
[2023-03-05T20:10:40.158+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-03-05T20:10:40.173+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): results_upload> on 2023-03-05 20:10:38.818733+00:00
[2023-03-05T20:10:40.178+0000] {standard_task_runner.py:55} INFO - Started process 10910 to run task
[2023-03-05T20:10:40.181+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'formula1_upload_to_s3', 'results_upload', 'manual__2023-03-05T20:10:38.818733+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/s3_upload_dag.py', '--cfg-path', '/tmp/tmpzxlsj99f']
[2023-03-05T20:10:40.181+0000] {standard_task_runner.py:83} INFO - Job 15: Subtask results_upload
[2023-03-05T20:10:40.230+0000] {task_command.py:388} INFO - Running <TaskInstance: formula1_upload_to_s3.results_upload manual__2023-03-05T20:10:38.818733+00:00 [running]> on host a4cedc34403c
[2023-03-05T20:10:40.307+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=tem109
AIRFLOW_CTX_DAG_ID=formula1_upload_to_s3
AIRFLOW_CTX_TASK_ID=results_upload
AIRFLOW_CTX_EXECUTION_DATE=2023-03-05T20:10:38.818733+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-05T20:10:38.818733+00:00
[2023-03-05T20:10:40.316+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-03-05T20:10:40.604+0000] {s3_upload_dag.py:104} INFO - Saved results data
[2023-03-05T20:10:40.616+0000] {base.py:73} INFO - Using connection ID 's3_conn' for task execution.
[2023-03-05T20:10:40.617+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='s3_conn', conn_type='aws') credentials retrieved from login and password.
[2023-03-05T20:10:41.505+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-03-05T20:10:41.513+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=formula1_upload_to_s3, task_id=results_upload, execution_date=20230305T201038, start_date=20230305T201040, end_date=20230305T201041
[2023-03-05T20:10:41.556+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-03-05T20:10:41.569+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
